### PR TITLE
feat: Develop Earn Page Layout Component

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

File: packages/web/src/components/earn/pool-list-table/PoolListTable.stories.tsx
Comment (jinoosss) [inline packages/web/src/components/earn/pool-list-table/PoolListTable.stories.tsx:1]: Failed to load storybook.

File: packages/web/src/components/earn/pool-list/PoolList.stories.tsx
Comment (jinoosss) [inline packages/web/src/components/earn/pool-list/PoolList.stories.tsx:1]: Failed to load storybook.

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.stories.tsx [hunk 1]
```diff
@@ -22,4 +22,6 @@ const Template: ComponentStory<typeof MyPositionCardList> = args => (
 export const Default = Template.bind({});
 Default.args = {
```
Comment (jinoosss) [inline packages/web/src/components/common/my-position-card-list/MyPositionCardList.stories.tsx:23]: It has fewer properties set.

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx
Comment (jinoosss) [inline packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx:1]: Showing components by comparing their windowSize is render intensive.
It's better to use conditional rendering with a type or boolean that doesn't change much in value.

File: packages/web/src/components/earn/incentivized-pool-card-list/IncentivizedPoolCardList.tsx [hunk 2]
```diff
@@ -7,55 +7,54 @@ import { type PoolListProps } from "@containers/incentivized-pool-card-list-cont
...
+      {!isFetched &&
+        Array.from({ length: 8 }).map((_, idx) => (
+          <span
+            key={idx}
+            className="card-skeleton"
+            css={skeletonStyle("100%", SHAPE_TYPES.ROUNDED_SQUARE)}
+          />
+        ))}
+    </PoolListWrapper>
+    {windowSize > 1000 ? (
```
Comment (jinoosss) [inline packages/web/src/components/earn/incentivized-pool-card-list/IncentivizedPoolCardList.tsx:45]: Showing components by comparing their windowSize is render intensive.
It's better to use conditional rendering with a type or boolean that doesn't change much in value.

File: packages/web/src/components/pool/my-position-card/MyPositionCard.stories.tsx
Comment (jinoosss) [inline packages/web/src/components/pool/my-position-card/MyPositionCard.stories.tsx:1]: Failed to load storybook.

File: packages/web/src/components/pool/pool-pair-info-content/PoolPairInfoContent.stories.tsx [hunk 3]
```diff
@@ -2,17 +2,34 @@ import React from "react";
...
+const wrapper = () => css`
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  margin-top: 50px;
+`;
+
+const contentWrap = () => css`
+  width: 800px;
```
Comment (jinoosss) [inline packages/web/src/components/pool/pool-pair-info-content/PoolPairInfoContent.stories.tsx:34]: The layout is broken in the storybook.

File: packages/web/src/components/pool/pool-pair-information/PoolPairInformation.stories.tsx
Comment (jinoosss) [inline packages/web/src/components/pool/pool-pair-information/PoolPairInformation.stories.tsx:1]: Failed to load storybook.

**Review by jinoosss — COMMENTED** [review]
LGTM !
I left a few comments.

File: packages/web/src/components/earn/pool-list-table/PoolListTable.stories.tsx
Comment (juntaepark) [reply packages/web/src/components/earn/pool-list-table/PoolListTable.stories.tsx:1]: sorry it works again.

File: packages/web/src/components/earn/pool-list/PoolList.stories.tsx
Comment (juntaepark) [reply packages/web/src/components/earn/pool-list/PoolList.stories.tsx:1]: sorry it works again.

File: packages/web/src/components/earn/earn-my-positions-content/EarnMyPositionsContent.stories.tsx [hunk 4]
```diff
@@ -22,8 +22,9 @@ const Template: ComponentStory<typeof EarnMyPositionsContent> = args => (
       <MyPositionCardList
         list={dummyPositionList}
         isFetched={true}
-        windowSize={500}
         currentIndex={1}
+        routeItem={() => {}}
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/components/earn/earn-my-positions-content/EarnMyPositionsContent.stories.tsx:26]: corrected.

File: packages/web/src/components/pool/pool-pair-info-content/PoolPairInfoContent.stories.tsx (hunk 3 above)
Comment (juntaepark) [reply packages/web/src/components/pool/pool-pair-info-content/PoolPairInfoContent.stories.tsx:34]: corrected.

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.stories.tsx (hunk 1 above)
Comment (juntaepark) [reply packages/web/src/components/common/my-position-card-list/MyPositionCardList.stories.tsx:23]: added.

File: packages/web/src/components/earn/incentivized-pool-card-list/IncentivizedPoolCardList.tsx (hunk 2 above)
Comment (juntaepark) [reply packages/web/src/components/earn/incentivized-pool-card-list/IncentivizedPoolCardList.tsx:45]: Good idea. I fixed it by rendering with boolean value.

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx
Comment (juntaepark) [reply packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx:1]: Good idea. I fixed it by rendering with boolean value.

**Review by jinoosss — APPROVED** [review]
Approve !

**Pull request merged**